addons/project/static/src/components/subtask_one2many_field/subtask_list_renderer.js

```diff
@@ -4,10 +4,22 @@ import { useService } from "@web/core/utils/hooks";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
 import { ListRenderer } from '@web/views/list/list_renderer';
 
+const { useEffect } = owl;
+
 export class SubtaskListRenderer extends ListRenderer {
     setup() {
         super.setup();
         this.dialog = useService("dialog");
+        useEffect(
+            () => this.focusName(this.props.list.editedRecord),
+        );
```

Comment:
We could add `this.props.list.editedRecord` in the dependencies of that `useEffect`.

```suggestion
        useEffect(
            () => this.focusName(this.props.list.editedRecord),
            () => [this.props.list.editedRecord]
        );
```